includes/wikia/DefaultSettings.php

```diff
@@ -351,6 +351,8 @@
 $wgAutoloadClasses['FormBuilderService']  =  $IP.'/includes/wikia/services/FormBuilderService.class.php';
 $wgAutoloadClasses['LicensedWikisService']  =  $IP.'/includes/wikia/services/LicensedWikisService.class.php';
 $wgAutoloadClasses['ArticleQualityService'] = $IP.'/includes/wikia/services/ArticleQualityService.php';
+$wgAutoloadClasses['ConsulService'] = $IP.'/includes/wikia/services/ConsulService.class.php';
+$wgAutoloadClasses['ExternalServicesQueryService'] = $IP.'/includes/wikia/services/ExternalServicesQueryService.class.php';
```

Comment:
I think this code should reside in [`app/lib/Wikia`](https://github.com/Wikia/app/tree/dev/lib/Wikia/src) instead of `app/includes/wikia/services`. That way, it will comply with the [PHP autoloading standards](http://www.php-fig.org/psr/psr-4/) and you won't have to do the manual autoloading.

For the name/ns, I would suggest `Wikia/Util/Consul` or `Wikia/Service/Util/Consul`.